File: answers/Fab-Gaurav/Day 13/Answer_2.cpp
#include<bits/stdc++.h>
using namespace std;

int palindrome(int arr[], int begin, int end){
    if (begin >= end){
        return 1;
    }
    if (arr[begin] == arr[end]){
        return palindrome(arr, begin + 1, end - 1);
    }
    else{
        return 0;
    }
}

int main(){
    int a[50],x;
    cout<<"Enter the number of elements in array : ";
    cin>>x;
    cout<<"Enter the elements of array : ";
    for(int i=1; i<x; i++){
        cin>>a[x];
    }
    
    int n = sizeof(a) / sizeof(a[0]);
    
    if (palindrome(a, 0, n - 1) == 1)
        cout << "PALINDROME";
    else
        cout << "NOT PALINDROME";
    return 0;
}

File: answers/Shobhit403/DAY 26/Answer2.py
size= int(input("Size of array: "))
arr=input("Enter the elements of array: ").split()
t=0
arr[0]=int(arr[0])
for i in range(1,size):
    arr[i]=int(arr[i])
    while arr[i]<arr[i-1]:
        t+=1
        arr[i]+=1
print(t)
